environments/overlays/cicd/pipeline-bot.yaml

```
@@ -0,0 +1,22 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: build-bot
+secrets:
+  - name: github-cred
+  - name: quay-cred
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  creationTimestamp: null
+  name: cicd-admin
```

Same, this can be something like
```suggestion
  name: pipeline-credentials-rolebinding
```